### PR TITLE
fix(navitem): pass unspecified props to button

### PR DESCRIPTION
**Pull request by jaketrent**
yeeehaa, pass onClicks

### CODE REVIEW

File: packages/navitem/src/react/context.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface ContextValue {
   selected: boolean
   // eslint-disable-next-line camelcase
   UNSAFE_stylesFor?: Partial<Record<AllowedSelectors, CSSProperties>>
+  rest?: Record<string, unknown>
```
Comment (danethurber) [inline packages/navitem/src/react/context.tsx:21]: why a named prop for rest?

**Review by danethurber — APPROVED** [review]
works for me. left one comment

File: packages/navitem/src/react/context.tsx (hunk 1 above)
Comment (jaketrent) [reply packages/navitem/src/react/context.tsx:21]: It's not a prop.  It's on context.  Context is the way that we're pass stuff to inner elements now.

Comment (danethurber) [reply packages/navitem/src/react/context.tsx:21]: ah...got it 👍

**Pull request merged**